docs/source/index.rst

```diff
@@ -1,13 +1,44 @@
-.. include:: about.md
-   :parser: myst_parser.sphinx_
+
+.. include:: substitutions.rst
+
+
+|graphnet|\ GraphNeT is an open-source Python framework aimed at providing high quality, user friendly, end-to-end functionality to perform reconstruction tasks at neutrino telescopes using deep learning. |graphnet|\ GraphNeT makes it fast and easy to train complex models that can provide event reconstruction with state-of-the-art performance, for arbitrary detector configurations, with inference times that are orders of magnitude faster than traditional reconstruction techniques.
```

Comment:
> GraphNeT is an open-source Python framework aimed at providing high quality, user friendly, end-to-end functionality to perform reconstruction tasks at neutrino telescopes using deep learning.

⬇️ 

GraphNeT is an open-source Python framework aimed at providing high-quality, user-friendly, end-to-end functionality for performing reconstruction tasks at neutrino telescopes using deep learning.